build/scripts/unzip.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Unzip reports a warning that zip contains backslashes and returns exit code 1 
+# even though everything is good. Ignore exit code 1.
```

Comment:
Yes, as mentioned above, the problem is powershell zip. Also, I don't think everything's necessarily fine with the unzipped file. I've seen files incorrectly unzipped before.